.github/workflows/ci.yml

```diff
@@ -35,10 +35,12 @@ jobs:
       id: buildx
       uses: docker/setup-buildx-action@v2
       with:
-        driver: docker
+        driver-opts: network=host
+    - name: Log in to registry
+      run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u $ --password-stdin
```

Comment:
Took this from https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions#upgrading-a-workflow-that-accesses-ghcrio.